cli/internal/fs/path.go

```diff
@@ -150,6 +150,16 @@ func (ap AbsolutePath) Symlink(target string) error {
 	return os.Symlink(target, ap.asString())
 }
 
+// Readlink implements os.Readlink for an absolute path
+func (ap AbsolutePath) Readlink() (string, error) {
+	return os.Readlink(ap.asString())
+}
+
+// Link implements os.Link(target, ap) for absolute path
+func (ap AbsolutePath) Link(target string) error {
+	return os.Link(ap.asString(), target)
```

Comment:
The argument order doesn't match the comment.

Reply:
👍🏼 fixed here and in the source where I copied it from 😄